backup_and_restore/application_backup_and_restore/installing/oadp-installing-operator.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-installing-operator-doc"]
+= Installing the OADP Operator
+include::_attributes/common-attributes.adoc[]
+:context: installing-oadp-operator
+
+toc::[]
+
+You install the OpenShift API for Data Protection (OADP) Operator on {product-title} {product-version} by using Operator Lifecycle Manager (OLM).
+
+The OADP Operator installs link:https://{velero-domain}/docs/v{velero-version}/[Velero {velero-version}].
```

Comment:
Since what dictates the version in the docs is OCP and not OADP, I would put a link here to the [OADP/velero version matrix](https://61927--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/troubleshooting.html#velero-obtaining-by-downloading_oadp-troubleshooting) (or even move the version matrix to here)

Reply:
I have checked the `velero-version` variable usage (with `grep -Iinr 'velero-version' .`) and I think would be better if we kill it. It seems hard to maintain it. The Velero version OADP installs changes depending in the OADP version (we can select OCP version in the docs, but not OADP).

I suggest highlighting the Velero/OADP version matrix and pointing to it where needed (for places that do not depend on a exact velero version, maybe use `main` version).

Since this would be a relative big change, I would like to see others opinions first

Reply:
Moving this table from troubleshooting, is something like this what you meant?

I think it might be better on a separate page

Reply:
I think maybe putting it under installing (that is the first time that velero versions is cited) would be good (and it can be a separate page under the installing section yes, for example)

Putting that info in just one place I believe is good for updating in the future. Today I saw it in 2 places (https://61927--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/oadp-installing-operator.html and https://61927--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/troubleshooting.html)

Reply:
Yeah i have not finished so there is still some duplication. :(

Reply:
> I have checked the `velero-version` variable usage (with `grep -Iinr 'velero-version' .`) and I think would be better if we kill it. It seems hard to maintain it. The Velero version OADP installs changes depending in the OADP version (we can select OCP version in the docs, but not OADP).
> 
> I suggest highlighting the Velero/OADP version matrix and pointing to it where needed (for places that do not depend on a exact velero version, maybe use `main` version).
> 
> Since this would be a relative big change, I would like to see others opinions first

We do NOT need to reference the Velero version in these docs :)

Reply:
I see what you are referring to now. 
https://61927--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/oadp-installing-operator.html#velero-product-version

![Screenshot from 2023-07-11 15-06-57](https://github.com/openshift/openshift-docs/assets/138787/90ac7beb-86c0-49a1-baf8-c19a3104b502)

@anarnold97 ok... for your reference, we do NOT update the velero y stream in an OADP y stream.
So ..
OADP 1.0.x will always be on velero 1.7.x
OADP 1.1.x will always be on velero 1.9.x
OADP 1.2.x will always be on velero 1.11.x

The table is informative for customers so let's leave it in. The alternative to save future doc updates would be to simply have the customer execute `velero version`.  This is nicer. Classy even :)